components/ad/smartbanner/src/index.js

```diff
@@ -17,8 +18,12 @@ export default class AdSmartbanner extends Component {
   }
 
   render () {
+    const className = cx('sui-AdSmartbanner', {
+      'sui-AdSmartbanner-static': this.props.static
```

Comment:
What do you think about defining the static class as a modifier? something like `is-static`.